src/test/java/com/znsio/sample/e2e/businessLayer/indigo/VoucherBL.java

```diff
@@ -0,0 +1,75 @@
+package com.znsio.sample.e2e.businessLayer.indigo;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.entities.Platform;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Randomizer;
+import com.znsio.sample.e2e.entities.INDIGO_VOUCHER_CONTEXT;
+import com.znsio.sample.e2e.screen.indigo.LandingPageScreen;
+import com.znsio.sample.e2e.screen.indigo.PromoScreen;
+import org.apache.log4j.Logger;
+import org.assertj.core.api.SoftAssertions;
+
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertThat;
+
+public class VoucherBL {
+    private static final Map<String, String> data = new LinkedHashMap<>();
+    private static final Logger LOGGER = Logger.getLogger(VoucherBL.class.getName());
+    public final TestExecutionContext context;
+    private final SoftAssertions softly;
+    Map<String, String> testData = Runner.getTestDataAsMap(System.getProperty("user.name"));
```

Comment:
why do we have testData here?

Reply:
Unused Line , Forgot to remove. removing it.